### DISCUSSION TITLE
Collapse-arrow problem

### DISCUSSION THREAD

**Discussion by abcbrookie** [saadeghi/daisyui#2005] [Q&A]
For whatever reason the collapse arrow is appearing as a diamond shape on mobile safari. The animation is also really clunky when expanding. Everything works fine on a scaled down desktop window, only experiencing the issue on mobile. 

https://github.com/saadeghi/daisyui/assets/35648646/90e6e657-db60-4439-9dde-baaebb939cde

**Comment by saadeghi** [comment]
Do you see this issue on daisyUI website as well?

Are you using `@tailwindcss/forms`?  
https://github.com/saadeghi/daisyui/issues/1431#issuecomment-1347822839

**Reply by abcbrookie** [reply]
Thanks for getting back ! 

The problem isn't present on the daisyUI website. 

I am not using `@tailwindcss/forms`.

These are my dependencies: 
<img width="260" alt="Screenshot 2023-06-16 at 2 32 50 PM" src="https://github.com/saadeghi/daisyui/assets/35648646/adb42bcd-2d44-4bd9-b493-eee6cf0e8d33">

Here is how I'm implementing the collapse element: 
```
        <div class="md:hidden collapse collapse-arrow w-full rounded-none ">

            <input type="checkbox" class="min-h-0 peer" />

            <x-banner class="bg-or-green-500 p-0 min-h-0 collapse-title">
                <span class="font-normal">
                    Articles - {{ $category == '' ? 'Recently Posted' : $category }}
                </span>
            </x-banner>

            <div class="collapse-content [input:checked~&]:mb-4 ">
                <div class="w-full">
                    @livewire('search-articles')
                </div>
                <x-articles.category-list />
            </div>

        </div>
```

The arrow showing up as a diamond happens only on mobile. But the animation lag happens on desktop Safari as well as seen in the attached video. 

https://github.com/saadeghi/daisyui/assets/35648646/7e692b8d-8c8f-4e2f-b3fc-5f9f90e2fec6
